CMSgov/mint-app
# [EASI-4080] GrahphQL Conversion

# EASI-4080


## Changes and Description

- Change 1
- Change 2



## How to test this change

1. `scripts/dev gql`
2. Bring up app and go through the process of creating task list and ensure nothing breaks
3. Verify no failures in tests 



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
